dockerfile/README.md

```diff
@@ -1,46 +1,66 @@
+>Here, only port 9443 (HTTPS servlet transport) has been mapped to a Docker host port.
+You may map other container service ports, which have been exposed to Docker host ports, as desired.
 
-* Docker run:
-    - Run the Identity Server 5.3.0 Docker container as follows;
-        + `docker run -it -p 9443:9443 wso2is:5.3.0`
+##### 6. Accessing management console.
+- To access the management console, use the docker host IP and port 9443.
+    + `https:<DOCKER_HOST>:9443/carbon`
+    
+>In here, <DOCKER_HOST> refers to hostname or IP of the host machine on top of which containers are spawned.
 
-* Access management console:
-    -  To access the management console, use the docker host IP and port 9443.
-        + `https:<DOCKER_HOST_IP>:9443/carbon`
 
 ## How to update configurations
-The configurations will be maintained on the Docker Host machine and volume mounted to the container.
+Configurations would lie on the Docker host machine and they can be volume mounted to the container. <br>
+As an example, steps required to change the port offset using `carbon.xml` is as follows.
 
-As an example, the steps required to change the port offset in `carbon.xml` is as follows:
+##### 1. Stop the Identity Server container if it's already running.
+In WSO2 Identity Server 5.3.0 product distribution, `carbon.xml` configuration file <br>
+can be found at `<DISTRIBUTION_HOME>/repository/conf`. Copy the file to some suitable location of the host machine, <br>
+referred to as `<SOURCE_CONFIGS>/carbon.xml` and change the offset value under ports to 1.
 
-* Stop the Identity Server container if it's already running.
+##### 2. Grant read permission to `other` users for `<SOURCE_CONFIGS>/carbon.xml`
+```
+chmod o+r <SOURCE_CONFIGS>/carbon.xml
+```
 
-* Create the required config changes in the host machine
-    - Extract the `wso2is-5.3.0.zip` file located in `DOCKERFILE_HOME/files`.
-        + Navigate to `<DOCKERFILE_HOME>/files` directory
-        + `unzip -q wso2is-5.3.0.zip`
-    - Change the port offset in `carbon.xml` file located in `DOCKERFILE_HOME/files/wso2is-5.3.0/repository/conf/` directory.
-    - Grant write permission to the `DOCKERFILE_HOME/files/wso2is-5.3.0/repository/conf/` directory on the host machine to `other` users;
-        + `sudo chmod o+w -R DOCKERFILE_HOME/files/wso2is-5.3.0/repository/conf`
+##### 3. Run the image by mounting the file to container as follows.
+```
+docker run 
+-p 9444:9444
+--volume <SOURCE_CONFIGS>/carbon.xml:<TARGET_CONFIGS>/carbon.xml
+wso2is:5.3.0
+```
 
-* Run the docker container by mounting the config directory (`DOCKERFILE_HOME/files/wso2is-5.3.0/repository/conf/`) of the host machine.
-    - Navigate to `<DOCKERFILE_HOME>` directory.
-    - `docker run -it --mount type=bind,source=${PWD}/files/wso2is-5.3.0/repository/conf,target=/home/wso2user/wso2is-5.3.0/repository/conf wso2is-5.3.0`
+>In here, <TARGET_CONFIGS> refers to /home/wso2carbon/wso2is-5.3.0/conf folder of the container.
```

Comment:
/home/wso2carbon/wso2is-5.3.0/conf should be changed to /home/wso2carbon/wso2is-5.3.0/repository/conf